common/values.go

```diff
@@ -87,9 +89,18 @@ const (
 	SecretKind         = "Secret"
 	ServiceKind        = "Service"
 	ServiceAccountKind = "ServiceAccount"
+	ClusterRoleKind    = "ClusterRole"
+	
 )
 
 // Commnds
 const (
 	LogLevel = "--loglevel"
 )
+
+// annotations
+const (
+	// AnnotationOpenShiftServiceCA is the annotation on services used to
+	// request a TLS certificate from OpenShift's Service CA for AutoTLS
+	AnnotationOpenShiftServiceCA = "service.beta.openshift.io/serving-cert-secret-name"
```

Comment:
since this is a map key i think we should have this in `keys.go`

Reply:
There is already a `ServiceBetaOpenshiftKeyCertSecret` key in `common/keys.go` with same value. Using it instead of adding a new one.